src/components/Article.jsx

```diff
@@ -1 +1,29 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+  dateISO = date.toISOString().slice(0, 10),
+  dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  }),
```

Comment:
I think you should move dateISO & dateLocaleString to the const from here. Because these are not input parameters, but variables that you use inside function.